lib/SPIRV/libSPIRV/SPIRVModule.cpp

```diff
@@ -1069,15 +1078,69 @@ SPIRVValue *SPIRVModuleImpl::addNullConstant(SPIRVType *Ty) {
 
 SPIRVValue *SPIRVModuleImpl::addCompositeConstant(
     SPIRVType *Ty, const std::vector<SPIRVValue *> &Elements) {
+  SPIRVConstantComposite *Res;
+  const size_t MaxNumElements = 65535 - 3;
```

Comment:
Magic numbers again, please outline those into some constants which are shared across the codebase

Reply:
Done as you suggested.